BUILDING.md

```diff
@@ -41,6 +41,21 @@ To run the tests:
 $ make test
 ```
 
```

Comment:
@@ -41,6 +41,21 @@ To run the tests: 
41    $ make test  41    $ make test  
42    ```  42    ```  
43      43      
  44   +To run the native module tests:  
  45   +  
  46   +```text  
  47   +$ make test-addons  
  48   +```  
  49   +  
  50   +To run the npm test suite:  
  51   +  
  52   +*note: to run the suite on node v4 or earlier you must first*  
  53   +*run `make install`*